README.md

```diff
@@ -1 +1,25 @@
 # hw01
+
+## About Me
+My name is Ming Cui. I received my master's degree in business administration from University of Science and Technology of China. I am working on my master's in social sciences at UChicago.
+
+## Why this course?
+**Computational tools** are trending. You may find
+
+1. Computational Neuroscience
+1. Computational Biology
+1. Computational Sociology
+1. Computational Psychology
+1. Computational Economics
+
+and more...
+
+As a learner in social sciences, it is really the time to study computational methods.
+
+## Things I have learned so far
+How to use Git workflow. I also found `<fork>` an interesting term. You may find [this website](https://www.atlassian.com/git/tutorials/comparing-workflows) useful for understanding `<fork>`.
+
+## What is next?
+Week 1 is pretty informative. I cannot wait for the incoming lectures.
+
+![](https://images.unsplash.com/photo-1555323912-3526a765162b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=600&q=60)
```

Comment:
Great job including a photo, maybe next time you can include some alternative text for the photo in the brackets as well!

Reply:
+1 alt text is great for accessibility on the web